docs/api/web-contents.md

```diff
@@ -1,8 +1,9 @@
 # webContents
 
+> Render and control web pages using lifecycle events.
```

Comment:
"using lifecycle events" is probably redundant, since users don't have to use events of WebContents to be able to control web pages..